admin_guide/allocating_node_resources.adoc

```diff
@@ -11,76 +11,69 @@
-on the effects of these settings, see xref:computing-allocated-resources[Computing Allocated Resources].
-====
+Use caution when you tune your cluster with resource limits and enforcing limits with evictions. Enforcing `system-reserved` limits can lead to starving critical system services from CPU or killing the critical system services by the OOM killer.
+
+In most cases, tuning resource allocation is performed by making an adjustment and then monitoring the cluster performance with a production-like workload. That process is repeated until the cluster is stable and meets service-level agreements.
+
+For more information on the effects of these settings, see xref:computing-allocated-resources[Computing allocated resources].
 
 [options="header",cols="1,2"]
 |===
 
 |Setting |Description
 
 |`*kube-reserved*`
-| Resources reserved for node resources, such as kubelet, the Docker daemon, SkyDNS, and kube-proxy. Default is none.
+| This setting is not used with {product-title}. Add the CPU and memory resources that you planned to reserve to `*system-reserved*` setting.
 
 |`*system-reserved*`
-| Resources reserved for the remaining system components. Default is none.
+| Resources that are reserved for the node components and system components. Default is none.
 |===
 
-You can see which services are controlled by `system-reserved`  with a tool such as libcgroup:
+You can view the services that are controlled by `system-reserved` with a tool such as libcgroup:
 
 [source,terminal]
 ----
-$ yum install libcgroup-tools 
+$ yum install libcgroup-tools
 $ lscgroup memory:/system.slice
 ----
 
-You can set these in the `*kubeletArguments*` section of the
+You can reserve resources in the `*kubeletArguments*` section of the
 xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node
-configuration map] by using a set of `<resource_type>=<resource_quantity>` pairs 
-(e.g., *cpu=200m,memory=512Mi*). Add the section if it does not already exist:
+configuration map] by adding a set of `<resource_type>=<resource_quantity>` pairs. For example, *cpu=500m,memory=1Gi* reserves 500 millicores of CPU and one gigabyte of memory.
 
-.Node Allocatable Resources Settings
+.Node allocatable resources settings
 ====
 [source,yaml]
 ----
 kubeletArguments:
-  kube-reserved:
-    - "cpu=200m,memory=512Mi"
   system-reserved:
-    - "cpu=200m,memory=512Mi"
+    - "cpu=500m,memory=1Gi"
 ----
 ====
 
+Add the `system-reserved` field if it does not exist.
+
 [NOTE]
 ====
 Do not edit the `node-config.yaml` file directly.
-==== 
+====
 
-To determine appropriate values for these settings, determine a node's resource usage using the node summary API.
-For more information, see xref:system-resources-reported-by-node[System Resources Reported by Node].
+To determine appropriate values for these settings, view a node's resource usage by using the node summary API. For more information, see xref:system-resources-reported-by-node[System resources reported by node].
```

Comment:
s/view a node's resource usage/view the resource usage of a node
(cuz ISG recommends avoiding possessives to reduce confusion)